AK/Time.h

```diff
@@ -139,7 +139,7 @@ class Time {
     }
     [[nodiscard]] static Time from_timespec(const struct timespec&);
     [[nodiscard]] static Time from_timeval(const struct timeval&);
-    [[nodiscard]] constexpr static Time min() { return Time(-0x8000'0000'0000'0000LL, 0); };
+    [[nodiscard]] constexpr static Time min() { return Time(-__INT64_MAX__ - 1LL, 0); };
```

Comment:
Can we use `INT64_MIN` here? We define it, and [this reference](https://www.cplusplus.com/reference/cstdint/) says that it should always exist.

Reply:
No because we don't include the right header (stdint.h) as this file is used in the Kernel and userspace. AK/Types doesn't define such a max

Reply:
Ah, right!

Reply:
I'll add a comment explaining this.